web/src/components/CategoryEntry.tsx

```diff
@@ -3,7 +3,7 @@ import { CategoryModel } from 'api-client'
 import iconPlaceholder from '../assets/IconPlaceholder.svg'
 import styled, { withTheme } from 'styled-components'
 import Highlighter from 'react-highlight-words'
-import normalizeSearchString from '../services/normalizeSearchString'
+import { normalizeSearchString } from '../utils/stringUtils'
```

Comment:
`string` instead of `stringUtils` would probably be enough as `utils`is already the dir. I guess we could also use helpers as in native.

Reply:
If we would use `helpers` you could also include stuff like `wrapDisplayName` there.

Reply:
Yeah, i think both would be valid. I had a look at the existing stuff and thought it makes most sense this way. If your comment is just a idea and not a must-do I prefer keeping the existing solution.